connector/gradle.properties

```diff
@@ -1,4 +1,4 @@
 group=org.oaebudt.edc
-version=0.11.1
-project.version=0.11.1
-edcGradlePluginsVersion=0.11.1
+version=0.12.0
+project.version=0.1.0
+edcGradlePluginsVersion=0.12.0
```

Comment:
what are these versions meant to? I would expect to find only the project version, the EDC version should be driven only by the version catalog